core/src/test/scala/com/rackspace/com/papi/components/checker/wadl/WADLCheckerSpec.scala

```diff
@@ -6996,7 +7195,7 @@ class WADLCheckerSpec extends BaseCheckerSpec with LogAssertions {
              XML.loadFile("src/test/resources/xsd/test-urlxsd.xsd"))
     When("the wadl is translated")
     val checker = builder.build (inWADL, TestConfig(false, false, true, true, true, 1,
-                                                    true, true, true, "XalanC",
+                                                    true, false, true, "XalanC",
```

Comment:
And again why were all of these changed to no longer do the XSD Grammar Transform?